src/components/Signup/Signup.js

```diff
@@ -1,5 +1,98 @@
 import React from "react";
-
+import { Navbar, Footer } from "../components";
+import { Link } from "react-router-dom";
 export default function Signup() {
-  return <div>Signup</div>;
+  return (
+    <div>
+      <Navbar />
+      <div className="d-flex flex-justify-center align-items-center">
+        <div className="card nav-yellow-shadow  login-container flex-column">
+          <div className="d-flex flex-justify-center">
+            <h3>Signup</h3>
+          </div>
+          <div className=" mt-2 ml-1 d-flex flex-column">
+            <label htmlFor="input-email" className="ml-2 input-label">
+              Email
+            </label>
+            <input
+              className="input input-login"
+              id="input-email"
+              placeholder="you@dreamstore.com"
+            />
```

Comment:
You can create an ` <Input>` component with `label` and `input` the pass the required value to it. This will reduce boilerplate codes.
You can use the same component for all the inputs, email, password, name

Reply:
Will surely do this while implementing authentication.
Thanks